src/script/common/c_content.cpp

```diff
@@ -1315,12 +1315,12 @@ void pushnode(lua_State *L, const MapNode &n)
 }
 
 /******************************************************************************/
-void warn_if_field_exists(lua_State *L, int table,
-		const char *name, const std::string &message)
+void warn_if_field_exists(lua_State *L, int table, const char *fieldname, 
```

Comment:
```suggestion
void warn_if_field_exists(lua_State *L, int table, const char *fieldname,
```

trailing whitespace removal